core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkspaceComponent.scala

```diff
@@ -348,6 +348,9 @@ trait WorkspaceComponent {
     ): ReadAction[Seq[Workspace]] =
       loadWorkspaces(findV2WorkspacesByIdsQuery(workspaceIds), attributeSpecs)
 
+    def listWorkspacesByNamespace(namespaceName: RawlsBillingProjectName): ReadAction[Seq[Workspace]] =
```

Comment:
There is already a method that does the same thing `listWithBillingProject`

Reply:
Thank you, I missed that! TBH, I don't really see the point of just a few methods being in `WorkspaceExtensions`.